util/trim.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package util
+
+import "strings"
+
+// TrimQuote returns a slice of the string s, with all leading
+// and trailing double or single quotes removed, as defined by Unicode.
+func TrimQuotes(s string) string {
```

Comment:
I forgot to push my comment yesterday. I suggest use a function to trim multiple chars if it's your intention. /cc: @shahzzzam 

https://github.com/Azure/acr-builder/blob/685dad4e79a13d96b87d0e80a86751e777ab23af/builder/builder.go#L381